package.json

```diff
@@ -58,7 +58,8 @@
     "@opentelemetry/resources": "^1.12.0",
     "@opentelemetry/sdk-node": "^0.38.0",
     "@opentelemetry/sdk-trace-node": "^1.12.0",
-    "@opentelemetry/semantic-conventions": "^1.12.0"
+    "@opentelemetry/semantic-conventions": "^1.12.0",
+    "next": "^13.4.4"
```

Comment:
This was only necessary for types but I wasn't sure how to get them otherwise, any advice would be appreciated